app/src/main/java/nl/tudelft/b_b_w/controller/KeyReader.java

```diff
@@ -0,0 +1,49 @@
+package nl.tudelft.b_b_w.controller;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+
+/**
+ * Class to read the private key from the file
+ */
+public class KeyReader {
```

Comment:
Because you have to care about the single responsibility principle:

http://www.oodesign.com/single-responsibility-principle.html